<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrmParameterTigerVNC</class>
 <widget class="QWidget" name="CFrmParameterTigerVNC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set TigerVNC parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Base">
      <attribute name="title">
       <string>Base</string>
      </attribute>
      <widget class="QCheckBox" name="cbEnable">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>72</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>38</y>
         <width>184</width>
         <height>28</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbPort">
          <property name="maximum">
           <number>65536</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>71</y>
         <width>219</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbShow">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../App/Viewer/Resource/Resource.qrc">
            <normaloff>:/image/EyeOn</normaloff>:/image/EyeOn</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../App/Viewer/Resource/Resource.qrc"/>
  <include location="../../../App/Viewer/Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
